src/listener.iced

```diff
@@ -1,5 +1,4 @@
 
-net = require 'net'
```

Comment:
To use this in react-native we can't have the packager pull in deps it doesn't have (like net), so instead we defer loading until its needed (which we never do since we have our own transport)